package.json

```diff
@@ -7,6 +7,7 @@
     "@testing-library/react": "^11.1.0",
     "@testing-library/user-event": "^12.1.10",
     "lodash.clonedeep": "^4.5.0",
+    "moment": "^2.29.1",
```

Comment:
Maybe we should use DayJS instead of moment, to reduce file size and to avoid mutability of datetime objects.
https://www.npmjs.com/package/dayjs
https://www.npmjs.com/package/react-dayjs